src/Stmt.cc

```diff
@@ -184,10 +185,50 @@ TraversalCode ExprListStmt::Traverse(TraversalCallback* cb) const
 
 static BroFile* print_stdout = 0;
 
+TableVal* get_string_set_from_vals (val_list* vals, ODesc* d)
+	{
+	ListVal* set = new ListVal(TYPE_STRING);
+	for ( int i = 0; i < vals->length(); i++ )
+		{
+		d->Clear();
+		Val* val = (*vals)[i];
+		val->Describe(d);
+		set->Append(new StringVal(d->Description()));
+		}
+
+	return set->ConvertToSet();
```

Comment:
Think `set` will end up being leaked here.  It needs to get `Unref()`d before returning (or better use an `IntrusivePtr`).

Speaking of leaks, mind trying to add a memory leak test that will hit this code path?  If you haven't added one before, it could be a good approach to add the test first, see it fail, then fix the leak and see it pass.